src/scanner/btc.go

```diff
@@ -368,11 +116,36 @@ func (s *BTCScanner) getBlockAtHeight(height int64) (*btcjson.GetBlockVerboseRes
 		return nil, err
 	}
 
-	return block, nil
+	return btcBlock2CommonBlock(block)
+
+}
+
+// btcBlock2CommonBlock convert bitcoin block to common block
+func btcBlock2CommonBlock(block *btcjson.GetBlockVerboseResult) (*CommonBlock, error) {
+	if len(block.RawTx) == 0 {
+		return nil, ErrBtcdTxindexDisabled
+	}
+	cb := CommonBlock{Hash: block.Hash, NextHash: block.NextHash, Height: block.Height}
+	cb.RawTx = make([]CommonTx, len(block.RawTx))
+	for _, tx := range block.RawTx {
+		cbTx := CommonTx{Txid: tx.Txid}
+		cbTx.Vout = make([]CommonVout, len(tx.Vout))
```

Comment:
Same here, must be `make([]CommonTx, 0, len(block.RawTx))` if you append to `Vout`, otherwise assign to `Vout[i]`